lib/nostrum/struct/component.ex

```diff
@@ -210,7 +225,7 @@ defmodule Nostrum.Struct.Component do
 
   Valid for [Buttons](#module-buttons)
   """
-  @type emoji :: Emoji.partial_button_emoji() | nil
+  @type emoji :: %{id: Emoji.id(), name: Emoji.name(), animated: Emoji.animated()} | nil
```

Comment:
Ahh, this will conflict with `master`, I've changed that to plain `map()` there, sorry ☹️